app/javascript/react/screens/App/Overview/screens/PlanWizard/PlanWizard.scss

```diff
@@ -9,3 +9,27 @@
 .plan-wizard-confirm-modal {
   margin-top: 140px;
 }
+
+.plan-wizard-vm-power-warning {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  text-align: left;
+}
+
+.wizard-pf-complete .plan-wizard-vm-power-warning {
+  margin-top: 20px;
+  justify-content: center;
+}
+
+.plan-wizard-vm-power-warning .pficon {
+  font-size: 24px;
+}
+
+.wizard-pf-complete .plan-wizard-vm-power-warning .pficon::before {
+  color: #ec7a08; /* overrides grey color from .blank-slate-pf */
```

Comment:
Can we please use the PF variable here :)

https://github.com/patternfly/patternfly/blob/master/src/sass/converted/patternfly/_color-variables.scss#L58

Reply:
Of course! Good catch, I forgot about those.